### DISCUSSION TITLE
Need help on windows install. thanks

### DISCUSSION THREAD

**Discussion by jajawunderbar** [CadQuery/cadquery#1179] [Q&A] [answered]
Installed cadquery by: curl -L -o miniforge.exe https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe
start /wait "" miniforge.exe /InstallationType=JustMe /RegisterPython=0 /NoRegistry=1 /NoScripts=1 /S /D=%USERPROFILE%\Miniforge3
:: Activate
cmd /K ""%USERPROFILE%/Miniforge3/Scripts/activate.bat" "%USERPROFILE%/Miniforge3""

Installed editor by unpacking zip.

Now how to let the editor know about cadquery under windows?
Because in the editor only works the default command, not the commands of the quickstart examples. 
Thanks, Mathias

Here the cmd log:

C:\Users\mathi\Downloads\CQ-editor-Windows>curl -L -o miniforge.exe https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 54.0M  100 54.0M    0     0  6271k      0  0:00:08  0:00:08 --:--:-- 7128k

C:\Users\mathi\Downloads\CQ-editor-Windows>start /wait "" miniforge.exe /InstallationType=JustMe /RegisterPython=0 /NoRegistry=1 /NoScripts=0 /S /D=C:\Users\mathi\Miniforge3

C:\Users\mathi\Downloads\CQ-editor-Windows>cmd /K ""C:\Users\mathi/Miniforge3/Scripts/activate.bat" "C:\Users\mathi/Miniforge3""

(base) C:\Users\mathi\Downloads\CQ-editor-Windows>

**Comment by lorenzncode** [comment, marked as answer]
Your command prompt shows `(base)`. This indicates you have conda installed and the base environment is active.   Next, install both cadquery and cq-editor to a new conda environment. 

The steps are listed here for cadquery: https://github.com/CadQuery/cadquery#cadquery-installation-via-conda

```
# Set up a new environment
conda create -n cadquery

# Activate the new environment
conda activate cadquery

# CadQuery development is moving quickly, so it is best to install the latest version from GitHub master
conda install -c conda-forge -c cadquery cadquery=master
```

Now cadquery is installed to a conda env named "cadquery".  You should see `(cadquery)` in the command prompt now instead of base.  At this point you could use cadquery as a Python library without the CQ-editor GUI.


Now install CQ-editor:

```
conda install -c conda-forge -c cadquery cq-editor=master
```

You can then launch cq-editor from the command prompt.

>Installed editor by unpacking zip.
>Now how to let the editor know about cadquery under windows?

You can install both cadquery and CQ-editor with conda.  I'd suggest to ignore the zip install since you have already installed conda.

**Reply by jajawunderbar** [reply]
Yes now it runs. Thank you very much!
